src/pages/main/Main.js

```diff
@@ -0,0 +1,118 @@
+import { useEffect, useState } from 'react';
+import Nav from '../../components/Nav/Nav';
+import Footer from './Footer';
+import SectionImg from './SectionImg';
+import MainButton from './MainButton';
+import './Main.scss';
+
+const Main = () => {
+  const [slideIndex, setSlideIndex] = useState(1);
+
+  const moveSlide = index => {
+    setSlideIndex(index);
+  };
+
+  useEffect(() => {
+    const timer = setInterval(() => {
+      if (slideIndex !== IMG_URLS.length) {
+        setSlideIndex(slideIndex + 1);
+      } else if (slideIndex === IMG_URLS.length) {
+        setSlideIndex(1);
+      }
+    }, 3000);
+    return () => {
+      clearInterval(timer);
+    };
+  }, [slideIndex]);
+
+  return (
+    <>
+      <Nav />
+      <div className="fullPageContainer">
+        <section id="first" className="mainSection">
+          {IMG_URLS.map(({ id, url }, index) => {
+            return (
+              <div
+                className={
+                  slideIndex === index + 1 ? 'slideImg active' : 'slideImg'
+                }
+                key={id}
+              >
+                <img src={url} alt="배경 이미지" />
+                <MainButton />
+              </div>
+            );
+          })}
+          <div className="containerDots">
+            {Array.from({ length: 3 }).map((empty, index) => (
```

Comment:
```suggestion
            {Array.from({ length: 3 }).map((_, index) => (
```
통상 안쓰는 paramter지만 순서 상 꼭 받아야 하는 경우에는 `_` 등으로 표현하기도 합니다, 이건 참고만 해주세요

Reply:
오 그런 방법이 있었군요... 감사합니다